pupil_src/shared_modules/video_capture/utils.py

```diff
@@ -478,6 +488,18 @@ def load_raw_data(path):
         raw_data.shape = (-1, 2)
         return np.asarray(raw_data, dtype=raw_data_dtype)
 
+    def load_worn_data(path):
+        if not (path and path.exists()):
+            return None
+
+        def confidence_from_uint8(uint8: int) -> float:
+            conf = uint8 / 255.0
+            conf = max(0.0, min(conf, 1.0))
+            return float(conf)
+
+        with open(path, "rb") as fh:
+            return [confidence_from_uint8(uint8) for uint8 in fh.read()]
```

Comment:
Please use `np.fromfile(str(path), "<u1")` to read the file and [`np.clip()`](https://numpy.org/doc/stable/reference/generated/numpy.clip.html) to clip the values. A conversion to float should not be required anymore after dividing by 255 as the result is converted automatically to float64.